src/template.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title><%= htmlWebpackPlugin.options.title %></title>
+</head>
+<body>
+    <header>
+        <h1>leaderboard</h1>
+    </header>
+    <main>
+        <section class="score">
+            <div class="head">
+                <h2>recent score</h2>
+                <button  id="reload">refresh</button>
+            </div>
+            <ul class="scores">
+              <li>name: 100</li>
+              <li>name: 100</li>
+              <li>name: 100</li>
+              <li>name: 100</li>
+              <li>name: 100</li>
+              <li>name: 100</li>
+              <li>name: 100</li>
+            </ul>
+        </section>
+
+        <section class="add">
+            <h2>Add your score</h2>
+            <div class="addscore">
+              <form action="" id="add">
+                <input type="text" placeholder="your name" id="name">
+                <input type="number" placeholder="your score" id="score">
+                <button>submit</button>
+              </form>
+            </div>
+            
+        </section>
```

Comment:
- [ ] Please align the `submit` button on the right side to match the wireframe design.

### Your design

![image](https://user-images.githubusercontent.com/83803414/215551516-374c94e2-1a93-4a10-9286-c19290436951.png)

### Wireframe design

![image](https://user-images.githubusercontent.com/83803414/215551619-96e60b9b-7e1b-4b50-91a3-427af72c2170.png)